src/postman.js

```
@@ -230,17 +236,36 @@
     window.addEventListener('message', function(e) {
         try {
             var message = Message.parse(e);
-            var client = postman.getClientByWindow(e.source);
-
-            if (!client)
-                return;
 
             switch (message.type) {
                 case 'req':
-                    client.handleRequest(message);
+                    var matchedClients = postman.getClientsByWindow(e.source);
```

Biome (2.5.6)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)